python-avd/pyavd/_eos_designs/structured_config/overlay/ip_security.py

```diff
@@ -118,3 +120,12 @@ def _set_key_controller(self: AvdStructuredConfigOverlayProtocol, profile_name:
         if self.shared_utils.is_wan_server:
             return
         self.structured_config.ip_security.key_controller.profile = profile_name
+
+    def _set_ipsec_parameters(self: AvdStructuredConfigOverlayProtocol) -> None:
+        """
+        Set parameters relevant to all ipsec connections.
+
+        Note: Duplicated in network_services/ip_security.py.
+        """
+        if self.inputs.ipsec_settings.bind_connection_to_interface:
+            self.structured_config.ip_security.connection_tx_interface_match_source_ip = True
```

Comment:
Since this is single line change may be we can move it in main function (ip_security).

Reply:
also the one in network_services/ip_security.py.

Reply:
agreed lets simplify for now